src/components/index.ts

```diff
@@ -2,6 +2,7 @@ export * from './Container';
 export * from './CTA';
 export * from './Header';
 export * from './Heading';
+export * from './Footer';
```

Comment:
🔧 Alphabetical order

```suggestion
export * from './Footer';
export * from './Header';
export * from './Heading';
```